src/lib.rs

```diff
@@ -170,6 +171,9 @@ cfg_if! {
         mod util_libc;
         // Keep std-only trait definitions for backwards compatiblity
         mod error_impls;
+    } else if #[cfg(target_os = "vxworks")] {
```

Comment:
See #105 and [my comment](https://github.com/rust-random/getrandom/pull/105#issuecomment-532492625), this won't properly include `error_impls` if `target_os = "vxworks"` and `feature = "std"`. The best bet is to just include it the big `cfg!` block above.